### PR TITLE
Add prow config for container-freezer

### PR DESCRIPTION
**Pull request by psschwei**
**What this PR does, why we need it**:

Add prow configs for https://github.com/knative-sandbox/container-freezer

**Which issue(s) this PR fixes**:

https://github.com/knative/community/issues/742

cc @julz @markusthoemmes

### CODE REVIEW

File: config/prod/prow/config_knative.yaml [hunk 1]
```diff
@@ -620,6 +620,11 @@ presubmits:
   - build-tests: true
   - unit-tests: true
   - integration-tests: false
+  knative-sandbox/container-freezer:
+  - build-tests: true
+  - unit-tests: true
+  - integration-tests: true
+  - go-coverage: false
```
Comment (julz) [inline config/prod/prow/config_knative.yaml:627]: not that Im personally a huge fan of coverage checkers, but any reason we're explicitly disabling the coverage checker here?

Comment (psschwei) [reply config/prod/prow/config_knative.yaml]: Mental lapse? I thought I had set all four of these to `true`, but I guess not...

Comment (psschwei) [reply config/prod/prow/config_knative.yaml]: @julz this has been fixed

**Pull request merged**